src/t8_forest/t8_forest_adapt.cxx

```diff
@@ -188,14 +190,17 @@ t8_forest_adapt (t8_forest_t forest)
   t8_locidx_t         el_coarsen;
   t8_locidx_t         num_el_from;
   t8_locidx_t         el_offset;
-  size_t              num_children, zz;
+  size_t              num_children, num_elements, zz, el_c;
```

Comment:
Give el_c a more meaningfull name, el_concidered_index for example

Reply:
Done.